src/openloco/input/mouse_input.cpp

```diff
@@ -928,6 +929,52 @@ namespace openloco::input
         }
     }
 
+    // 0x004C76A7
+    // regs.cx = (uint16_t)button;
+    // regs.ax = x;
+    // regs.bx = y;
+    static void stateScrollRight(const mouse_button button, const int16_t x, const int16_t y)
+    {
+        auto window = WindowManager::find(_dragWindowType, _dragWindowNumber);
+        if (window == nullptr)
+        {
+            input::state(input_state::reset);
+            return;
+        }
+
+        switch (button)
+        {
+            case mouse_button::released:
+            {
+                _ticksSinceDragStart += time_since_last_tick;
+                if (x != 0 || y != 0)
+                {
+                    _ticksSinceDragStart = 1000;
+                    ui::widget_t* widget = &window->widgets[_dragWidgetIndex];
+                    ui::scrollview::horizontalDragFollow(window, widget, _dragWidgetIndex, _dragScrollIndex, x);
+                    ui::scrollview::verticalDragFollow(window, widget, _dragWidgetIndex, _dragScrollIndex, y);
+                }
+
+                break;
+            }
+
+            case mouse_button::right_released:
+            {
+                input::state(input_state::reset);
+                if (_ticksSinceDragStart <= 500)
```

Comment:
I'd just remove this whole if. Add a note stating that in the assembly this set the x and y value and result was unused.